gemrb/GUIScripts/Console.py

```diff
@@ -95,7 +95,14 @@ def write(self, message):
 		if con:
 			sys.stdout = OutputCapture(stdout)
 
-		return eval(cmd)
+		locals = {} # we dont want to expose our locals
+		modend = cmd.find('.')
+		if modend > -1:
+			import importlib
+			importlib.invalidate_caches()
+			modname = cmd[0:modend]
+			locals[modname] = importlib.import_module(modname)
+		return eval(cmd, globals(), locals)
```

Comment:
Why is this commit needed? We already import our two modules for it both as modules and at least for the main one also with a wildcard import.

Reply:
this allows you to call functions inside guiscripts tho, like `GUICommonWindows.OpenPSTDeathWindow()`

it also hides the locals from the evaluation